src/qml/editorwidgets/DateTime.qml

```diff
@@ -52,8 +55,9 @@ Item {
 
         Image {
           source: Style.getThemeIcon("ic_clear_black_18dp")
-          anchors.left: parent.right
-          visible: main.currentValue !== undefined && config['allow_null']
+          anchors.right: parent.right
+          anchors.verticalCenter: parent.verticalCenter
```

Comment:
are both placements required? To me it looks like it should be either one or the other, since they can be contradicting?

Reply:
Why contradicting? One is vertical alignment and the other horizontal. If I make no `verticalCenter` it's on the right top corner...

Reply:
oops sorry